apps/passport-server/test/genericIssuance.spec.ts

```diff
@@ -438,7 +459,7 @@ t2,i1`,
       const bouncerChecksInAttendeeAgain = await requestCheckInPipelineTicket(
         edgeCityDenverPipeline.checkinCapability.getCheckinUrl(),
         ZUPASS_EDDSA_PRIVATE_KEY,
-        EdgeCityDenverBouncerTicket.email,
+        EdgeCityDenverBouncerTicket.user_email as string,
```

Comment:
in general i want to avoid casting in tests, perhaps a `expectToExist` call on `EdgeCityDenverBouncerTicket.user_email`, here and elsewhere that is relevant, could let you avoid using `as`

Reply:
These were left over from an earlier iteration of the API, which would sometimes return `null` if the user email wasn't known. The final implementation is more strictly types and so I can remove these.